purchase_report_custom_hls/report/purchase_order_report_templates.xml

```diff
@@ -32,4 +35,16 @@
             </div>
         </xpath>
     </template>
+    <!-- Ideally, we should cut this line out into another module. -->
+    <template
+        id="report_purchaseorder_document"
+        inherit_id="purchase_stock.report_purchaseorder_document"
+    >
+        <xpath expr="//t[@t-else='']/t[@t-set='information_block']/strong" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+        <xpath expr="//div[@t-if='o.picking_type_id and o.picking_type_id.warehouse_id']" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
```

Comment:
Edit this line just add HTML style attribute.
It doesn't work "invisible" attr in t-set tag.